docs/contributing/doc_guide.rst

```diff
@@ -1112,24 +1129,92 @@ Narrative documentation guidelines
-  occur when Sphinx is unable to interpret text as a Python object.
+This section contains suggestions about how to fix common documentation
+errors and warnings.
+
+Reference target not found
+--------------------------
+
+Warnings like ``py:obj reference target not found`` occur when Sphinx_
+attempts to interpret text as a Python object, but is unable to do so.
+For example, if a docstring includes ```y```, Sphinx will attempt
+to link to an object named ``y``. If there is no object named ``y``,
+then Sphinx will issue this warning, which gets treated like an error.
+
+If the text is meant to be an in-line code snippet, surround it with
+double backticks instead of single backticks.
+
+This warning may also occur when there is an extra space between a
+|role| and the argument it is intended to act on. For example, this
+warning would be fixed by changing ``:math: `y``` to
+``:math:`y```.
+
+When the text is meant to represent a code object like
+```astropy.units.Quantity```, then this warning usually indicates
+either a typo or an error in the namespace. For example, the warning
+resulting from ```astropy.Quantity``` can be resolved by changing it to
+```astropy.units.Quantity```.
 
-  - If the text is meant to be a code sample, make sure that the text is
-    in double back ticks.
+This warning sometimes occurs in the type specification of a |parameter|
+in a docstring. Sphinx attempts to link words in type specifications to
+code objects. Type lines are intended to provide concise information
+about allowed types, sizes, shapes, physical types, and default values
+of a parameter. To resolve this warning, first move information about
+the *meaning* of a parameter from the type specification into the
+parameter description that begins on the following line. To expand the
+list of allowed words or patterns in type specifications, add a regular
+expression to ``nitpick_ignore_regex`` in |docs/conf.py|_.
 
-  - If the text should refer to a code object, make sure that the full
-    namespace is provided correctly.
+Missing documentation pages for new modules
+-------------------------------------------
 
-  - If the text is in the type line for a parameter in a docstring, then
-    either:
+When a new module or subpackage is created, it is usually necessary to
+:ref:`create a stub file <api-static>` for it in |docs/api_static|_.
 
-    - Change the type line to include only Python types and words or
-      patterns defined in ``nitpick_ignore_regex`` in |docs/conf.py|_
-      (preferred), or
+Missing attribute errors
+------------------------
+
+An `AttributeError` may occur when an ``import`` statement is missing in
+a :file:`__init__.py` file.  For example, the error
+
+```
+AttributeError: module 'plasmapy.subpackage' has no attribute 'module'
+```
+
+will occur when :file:`plasmapy/subpackage/__init__.py` is missing
+:py:`import module`. Make sure that ``__all__`` contains ``"module"``
+as well.
```

Comment:
@pheuer — I believe this should resolve the documentation build `AttributeError` that is happening in #1274.  If it doesn't, please let me know and I can update this!

Reply:
Seems to have worked, since now we're on to another error :)